docs/services/slam/cartographer/_index.md

```diff
@@ -54,188 +81,380 @@ Follow the instructions below to set up the `cartographer` module on your robot:
+
+{{%expand "Click to view an example JSON configuration for updating an existing map" %}}
+
+See [Attributes](#attributes) for details.
+
+```json {class="line-numbers linkable-line-numbers"}
+{
   "modules": [
+    {
+      "type": "registry",
+      "name": "viam_rplidar",
+      "module_id": "viam:rplidar",
+      "version": "0.1.14"
+    },
     {
       "type": "registry",
       "name": "viam_cartographer",
       "module_id": "viam:cartographer",
       "version": "0.3.36"
+    }
+  ],
+  "services": [
+    {
+      "attributes": {
+        "config_params": {
+          "max_range_meters": "25",
+          "mode": "2d",
+          "min_range_meters": "0.2"
+        },
+        "camera": {
+          "name": "rplidar",
+          "data_frequency_hz": "5"
+        },
+        "enable_mapping": true,
+        "use_cloud_slam": true,
+        "existing_map": "${packages.slam_map.test-map-1}/internalState.pbstream"
+      },
+      "name": "slam",
+      "type": "slam",
+      "namespace": "rdk",
+      "model": "viam:slam:cartographer"
     },
     {
-      "type": "registry",
-      "name": "viam_rplidar",
-      "module_id": "viam:rplidar",
-      "version": "0.1.14"
+      "name": "Data-Management-Service",
+      "type": "data_manager",
+      "attributes": {
+        "tags": [],
+        "additional_sync_paths": [],
+        "sync_interval_mins": 0.1,
+        "capture_dir": ""
+      }
+    }
+  ],
+  "components": [
+    {
+      "namespace": "rdk",
+      "attributes": {},
+      "depends_on": [],
+      "service_configs": [
+        {
+          "attributes": {
+            "capture_methods": [
+              {
+                "disabled": false,
+                "method": "NextPointCloud",
+                "capture_frequency_hz": 5
+              }
+            ]
+          },
+          "type": "data_manager"
+        }
+      ],
+      "name": "rplidar",
+      "model": "viam:lidar:rplidar",
+      "type": "camera"
+    }
+  ],
+  "packages": [
+    {
+      "name": "test-map-1",
+      "version": "1697208847",
+      "package": "d1c224e8-483e-4cc7-980f-76b89d8fb507/test-map-1",
+      "type": "slam_map"
     }
   ]
 }
 ```
 
-To save your changes, click **Save config** at the bottom of the page.
+{{% /expand%}}
+
+#### Localize only
+
+In this mode, the cartographer-module on your robot executes the Cartographer algorithm itself locally.
 
-{{% /tab %}}
-{{% tab name="JSON Example" %}}
+The configuration is similar to the configuration for [updating an existing map](#update-existing-map), except instead of setting a `Data capture rate (Hz)` on the camera and movement sensor, you set a `Data polling rate (Hz)` on them, since the `cartographer-module` on your robot will poll the live LiDAR and IMU directly at these rates instead of the data being sent to the cloud.
```

Comment:
```suggestion
The configuration is similar to the configuration for [updating an existing map](#update-an-existing-map), except instead of configuring a `Data capture rate (Hz)` on the camera and movement sensor, set a `Data polling rate (Hz)` for both. The `cartographer-module` on your robot polls the live LiDAR and IMU directly at these rates, wheras the capture rate is only used when data is being sent to the cloud.
```